Documentation/usage/dlv.md

```diff
@@ -20,7 +20,7 @@ Pass flags to the program you are debugging using `--`, for example:
 
 ```
       --accept-multiclient   Allows a headless server to accept multiple client connections.
-      --api-version int      Selects API version when headless. (default 1)
+      --api-version int      Selects API version when headless (see Documentation/api/json-rpc/README.md). (default 1)
```

Comment:
I think people generally prefer not to open their browser and follow links when doing --help on the command line, so it might be helpful to include the info that is needed here.
What about:

Selects API version when headless. New clients should use v2. Can be reset via RPCServer.SetApiVersion. See Documentation/api/json-rpc/README.md.

Reply:
Done.